fdbrpc/tests/fdbrpc_bench.actor.cpp

```diff
@@ -0,0 +1,232 @@
+/*
+ * fdbrpc_bench.actor.cpp
```

Comment:
Can you add a description in the PR of what the end-goal is? There is already throughput measurement in https://github.com/spraza/foundationdb/blob/bf59b5434fe6aea429935868508347b623b250c8/documentation/tutorial/tutorial.actor.cpp#L397-L403, so I am wondering how this is different.

Reply:
Thanks for pointing to it. The motive for this particular one was (1) to just measure raw performance without anything else to quickly measure in controlled way .. the other example has for example map storing random key/values (2) also just separate out benchmarking in separate file (not that important but I good to have imo).